_common.scss

```
@@ -43,4 +43,8 @@
     .mat-tooltip {
         font-size: 12px;
     }
+
+    .mat-row {
+        height: 52px;
```

@huayunh do we want this to be a min-height, rather than an exact height?
```suggestion
        height: unset;
        min-height: 52px;
```

---

Can see it going both ways. Is the idea here to trigger a text wrap? If so, we shall ensure that the text actually wraps, and leave like a 16px padding to the top and bottom of the row content.

---

I like the min-height idea, agree with Huayun we should do a quick test.  I'm thinking if the test wraps, the height of the row will probably exceed 52px but would like to see it in the showcase demo.